usr/share/rear/conf/default.conf

```diff
@@ -2143,8 +2143,26 @@ GALAXY10_BACKUPSET=
 GALAXY10_Q_ARGUMENTFILE=
 ####
 
-####
-# BACKUP=TSM
+# BACKUP=GALAXY11
+##
+# Note: This is for CommVault 11
+#
+COPY_AS_IS_GALAXY11=( /opt/commvault/ /etc/CommVaultRegistry )
+COPY_AS_IS_EXCLUDE_GALAXY11=( "/opt/commvault/iDataAgent/jobResults/*" "/opt/commvault/Updates/*" "/opt/commvault/Base64/Temp/*" )
+# Use this default backup set. If left empty we will prompt the user at restore time for
+# the appropriate backup set to use. Could be set to a global organisation-wide default
+# in site.conf ...
+GALAXY11_BACKUPSET=
+# Use this argument file for all Commvault commands. You can place some general stuff here
+# and also use this file to store the logon credentials to Commvault. This file will be
+# automatically included in COPY_AS_IS
+GALAXY11_Q_ARGUMENTFILE=
```

Comment:
When GALAXY11_Q_ARGUMENTFILE cotains confidential data
it means the ReaR recovery system image file and each
medium made from it must be properly protected by the user.
Perhaps a similar info text about 'secrets'
as we have at other places in default.conf
(cf. BACKUP_PROG_CRYPT_KEY and SSH_UNPROTECTED_PRIVATE_KEYS)
should also be added to the GALAXY11_Q_ARGUMENTFILE description
to be on the safe side that we have properly informed the user?